templates/direct_menu.php

```diff
@@ -26,9 +26,13 @@
 
 	#navigation div ul {
 		position: relative;
-		margin-left: 170px;
+		padding-right:10px;
```

Comment:
what, was an less space between the icons and have not look very nice, so with 10 px by side, look's better, uniform and smooth.. try it out, be just 10 px, but this maks's it.. :)

best regards
blacky